tests/js/server/shell/shell-foxx-request-spec.js

```diff
@@ -541,6 +542,35 @@ describe('SyntheticRequest', function () {
         'https://www.example.com:9999/_db/bananas/another/place'
       );
     });
+    it('correctly handles unix sockets', function () {
```

Comment:
these tests must not be executed on windows - its got no unix domain sockets.
see `js/client/modules/@arangodb/testsuites/endpoints.js`
```
  opts['skipEndpointsUnix'] = (platform.substr(0, 3) === 'win');
```

Reply:
No, they're safe. These tests use the Foxx test-utils to create a dummy request. They're isolated unit tests, not integration tests.

Reply:
Disabling them on Windows is possible but it doesn't make a difference. They would run equally well on Windows as on Linux or MacOS because they don't actually involve unix sockets, they just ensure a request object with endpoint information is handled correctly.